day-to-day.md

```diff
@@ -51,10 +51,22 @@ Please also track your time out in Harvest.
 
 ### Flex time
 
-A clear system for time tracking allows for a flexible work schedule without feeling guilty. This allows you to mold your work hours to better fit your personal schedule. For example, if you need to get a haircut, and the best time to schedule that is 2pm on a Thursday, you can make up the hour you miss on another day (prior or subsequent). Its important to communicate your away from Keyboard time to your teammates.
+A clear system for time tracking allows for a flexible work schedule without feeling guilty. This allows you to mold your work hours to better fit your personal schedule. For example, if you need to get a haircut, and the best time to schedule that is 2pm on a Thursday, you can make up the hour you miss on another day (prior or subsequent). It's important to communicate to your teammates when you'll be away from the keyboard.
 
 > We trust that we'll make up our time in a timely manner
 
+### Professional Development
+
+Self-improvement is important, especially to help achieve personal goals for growth in our industry. Sometimes, we'll be on a project that doesn't directly nurture those goals. If you have urgent concerns about your current project, please let a teammate know so that we can work to [shift you to a new project](./project-placement.md).
+
```

Comment:
Add wording to highlight that some professional development should be billed to a current project.